tests/Behat/Context/Setup/FeatureContext.php

```diff
@@ -0,0 +1,168 @@
+<?php
```

Comment:
Does this file need to be here? It looks like somethings that's been copied from Sylius

Reply:
Yes, i'll remove it ;)